pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -175,10 +175,14 @@ private BrokerResponse executeSqlQuery(ObjectNode sqlRequestJson, HttpRequesterI
     }
   }
 
-  // TODO: Remove the SQL query options after releasing 0.11.0
-  private String constructSqlQueryOptions() {
-    return Request.QueryOptionKey.GROUP_BY_MODE + "=" + Request.SQL + ";" + Request.QueryOptionKey.RESPONSE_FORMAT + "="
-        + Request.SQL;
```

Comment:
can I delete this one now @Jackie-Jiang ^

Reply:
This should not be used anymore especially if legacy PQL code has been deleted

Reply:
@walterddr No, we cannot remove it before cutting the next release. Let's keep it for a while because the previous version server relies on this to correctly process the query

Reply:
sad.. ok. I will wait for 0.11 release cut before I merge this one then